Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -158,20 +145,23 @@ public struct WebView: UIViewRepresentable {
             }
             
             let baseURL = await parent.viewModel.baseURL
-            if !baseURL.isEmpty, !url.absoluteString.starts(with: baseURL) {
-                if navigationAction.navigationType == .other {
-                    return .allow
-                } else if navigationAction.navigationType == .linkActivated {
-                    await MainActor.run {
-                        UIApplication.shared.open(url, options: [:])
-                    }
-                } else if navigationAction.navigationType == .formSubmitted {
-                    return .allow
+            switch navigationAction.navigationType {
+            case .other:
+                return .allow
+            case .linkActivated:
+                await MainActor.run {
+                    UIApplication.shared.open(url, options: [:])
```

Comment:
It would be nice if you can check whether the link is openable or not before opening the link.

Reply:
done